src/omegga/plugin.ts

```diff
@@ -15,6 +15,12 @@ export interface IPluginJSON {
   formatVersion: number;
   omeggaVersion: string;
   emitConfig?: string;
+  dependencies?: {
+    [pluginName: string]: { optional?: boolean; repo?: string };
```

Comment:
possible to make this be `{ optional?, repo? } | string` where the `string` is the repo? kind of a shorthand